common/lib/capa/capa/templates/editamolecule.html

```diff
@@ -16,7 +16,7 @@
 
     <p id="answer_${id}" class="answer"></p>
 
-    <p class="status" aria-describedby="input_${id}">
+    <p class="indicator-container" aria-describedby="input_${id}">
```

Comment:
https://jsinput.sandbox.edx.org/courses/course-v1:jsinput+jsinput+jsinput/courseware/36ab1a99540048dfbb5b9af9218f5569/5d6e37b8d2da4609920859da9bfa57dc/3?activate_block_id=block-v1%3Ajsinput%2Bjsinput%2Bjsinput%2Btype%40vertical%2Bblock%4000d80aced5b845e888e0018e2bbd0942

Though note that this problem type is only working due to the temporary JSME commit.